openmdao/components/meta_model_unstructured.py

```diff
@@ -149,6 +166,11 @@ def _setup_vars(self, recurse=True):
         Return our inputs and outputs dictionaries re-keyed to use absolute variable names.
```

Comment:
looks like you removed the return value, making this description incorrect?

Reply:
yeah, Keith, just copy the description from the parent class.